spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/flyway/FlywayPropertiesTests.java

```diff
@@ -108,9 +109,7 @@ void expectedPropertiesAreManaged() {
 		// Handled by the conversion service
 		ignoreProperties(configuration, "baselineVersionAsString", "encodingAsString", "locationsAsStrings",
 				"targetAsString");
-		// Teams-only properties that we cannot detect as no exception is thrown and
 		// getters return null
```

Comment:
This line should be deleted as well please.

Reply:
Yep, good catch @wilkinsona .